.cirrus/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG CIRRUS_AWS_ACCOUNT=275878209202
 ARG JDK_VERSION=17
 FROM public.ecr.aws/docker/library/eclipse-temurin:11-jammy AS buildphase
 
-FROM ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j${JDK_VERSION}-m3.9-latest
+FROM ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base-pr-389:j${JDK_VERSION}-m3.9-latest
```

Comment:
Remove this change when the docker images containing digicert tooling are promoted to prod

Reply:
done